src/init/features/ailogic/index.spec.ts

```diff
@@ -0,0 +1,312 @@
+import * as prompt from "../../../prompt";
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as fs from "fs-extra";
+import * as init from "./index";
+import * as utils from "./utils";
+import { Setup } from "../..";
+import { Config } from "../../../config";
+
+describe("init ailogic", () => {
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe("askQuestions", () => {
+    let selectStub: sinon.SinonStub;
+    let inputStub: sinon.SinonStub;
+    let confirmStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      selectStub = sandbox.stub(prompt, "select");
+      inputStub = sandbox.stub(prompt, "input");
+      confirmStub = sandbox.stub(prompt, "confirm");
+    });
+
+    it("should populate ailogic featureInfo for android", async () => {
```

Comment:
Nonblocking: It would be great to cover the invalid input cases here too - there is some regex in this path to assert that the app name is formatted correctly that is not meaningfully covered by this.

Reply:
Greatly simplify the code, this is outdated